### PR TITLE
CIR-Admin-GlobalConfigration-Fields - Added blank files.

### PR DESCRIPTION
**Pull request by BlacyMacwan**

### CODE REVIEW

**Comment by mittalvyas1** [comment]
You need to commit project files as well

**Comment by BlacyMacwan** [comment]
>

**Comment by BlacyMacwan** [comment]
> You need to commit project files as well

I committed all .csproj files in 1st commit.

File: CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFieldsRepository.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using CIR.Common.Data;
+using CIR.Core.Interfaces.GlobalConfiguration;
+
+namespace CIR.Data.Data.GlobalConfiguration
+{
+	public class GlobalConfigurationFieldsRepository : IGlobalConfigurationFieldsRepository
+	{
+		#region PROPERTIES
+		private readonly CIRDbContext _cIRDbContext;
```
Comment (GohilAmit) [inline CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFieldsRepository.cs:9]: not use underscore

File: CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFieldsRepository.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+		#endregion
+
+		#region CONSTRUCTOR
+		public GlobalConfigurationFieldsRepository(CIRDbContext cIRDbContext)
+		{
+			_cIRDbContext = cIRDbContext;
```
Comment (GohilAmit) [inline CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFieldsRepository.cs:15]: please define argument null exception like this in constructor :
 cIRDbContext = context ??
                throw new ArgumentNullException(nameof(context));

File: CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFieldsRepository.cs (hunk 1 above)
Comment (BlacyMacwan) [reply CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFieldsRepository.cs]: Changes Applied.

File: CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFieldsRepository.cs (hunk 2 above)
Comment (BlacyMacwan) [reply CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFieldsRepository.cs]: Changes Applied.

**Review by GohilAmit — APPROVED** [review]

**Review by mittalvyas1 — APPROVED** [review]

**Review by jayk-patel-tatvasoft — APPROVED** [review]

**Pull request merged**